### ISSUE TITLE
Combination of L008 and L019 causes L008 triple count

### ISSUE THREAD

**Issue by jpy-git** [sqlfluff/sqlfluff#2001]
## Expected Behaviour
Running `sqlfluff lint` on 
```sql
SELECT
    col_a AS a
    ,col_b AS b
FROM foo;
```
should raise one L008 (Commas should be followed by a single whitespace unless followed by a comment.) and one L019 (Found leading comma. Expected only trailing.). 

## Observed Behaviour
L008 is raised 3 times:
![image](https://user-images.githubusercontent.com/80432516/143954771-8f495e53-ce7f-45af-9791-f32dc1c1f778.png)

## Steps to Reproduce
`sqlfluff lint` on the query above.

## Dialect
ansi

## Version
Include the output of `sqlfluff --version` along with your Python version
sqlfluff, version 0.8.2 (HEAD of main branch)

## Configuration
default config

**Comment by jpy-git** [comment]
Identified the cause of this, will open a PR
